File: src/misc/R-functions/hildebrand_rule.R
#' @title hildebrand_rule
#' 
#' @description Calculates Hildebrand rule for symmetry
#' @description H < 0.2 means symmetry
#' 
#' @param x The vector to analyze
#' @export
#' 
#' @return

hildebrand_rule <- function(x)
{
    (mean(x) - median(x))/sd(x)
}

File: src/statistics/hildebrand_rule.jl
export hildebrand_rule

"""
    hildebrand_rule(x)

Calculate Hildebrand rule for vector `x`.
If H < 0.2 then the vector `x` is symmetrical.

# Arguments

- `x::AbstractVector`

# Returns

- `h::Float64`
"""
function hildebrand_rule(x::AbstractVector)

    return (mean(x) - median(x)) ./ std(x)
    
end
